addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -254,7 +252,7 @@ def _l10n_es_edi_facturae_inv_lines_to_items(self, conversion_rate=None):
             totals['total_taxes_withheld'] += sum((abs(tax["tax_amount"]) for tax in taxes_withheld_computed))
 
             invoice_line_values.update({
-                'FileReference': self._l10n_es_edi_facturae_get_filename().split('.')[0][:20],
+                'FileReference': self.ref[:20] if self.ref else False,
```

Comment:
The change from name to ref is quite intrusive...

Reply:
In what way? It was incorrect in the first place so it has to be changed :sweat_smile: